e2e/test-component/scenarios/embedding-sdk/interactive-question.cy.spec.tsx

```diff
@@ -281,7 +281,7 @@ describeEE("scenarios > embedding-sdk > interactive-question", () => {
 
     getSdkRoot().within(() => {
       // Open the default summarization view in the sdk
-      cy.findByText("Summarize").click();
+      cy.findByText("1 summary").click();
```

Comment:
Is this because of the new default layout introduced in the other PR?

Reply:
Yep! If there's any issues we can sort them out in a separate PR